versions.yaml

```diff
@@ -194,7 +194,7 @@ assets:
   kernel:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v6.x/"
-    version: "v6.1.62"
+    version: "v6.6.32"
```

Comment:
Hi @fidencio !

just double-checking: it no longer need the https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/kernel/patches/6.1.x patches?